workflow/rocoto_funcs/prep_ic.py

```diff
@@ -71,6 +72,7 @@ def prep_ic(xmlFile, expdir, do_ensemble=False):
     <and>
       <and>
 {strneqs}
+<datadep age="00:03:00"><cyclestr offset="-{cyc_interval}:00:00">&COMROOT;/rrfs/v2.0.2/rrfsdet.@Y@m@d/@H/fcst/</cyclestr><cyclestr>mpasout.@Y-@m-@d_@H.00.00.nc</cyclestr></datadep>
```

Comment:
We may want to make this dependency more flexible as follows:

```
&COMROOT;/&NET;/&rrfs_ver;/&RUN;&WGF;.@Y@m@d/@H/fcst/
```

Reply:
I was going to make the same comment as @guoqing-noaa about making it more flexible with regard to version number.